docs/sig_qualityofservice.md

```diff
@@ -0,0 +1,38 @@
+---
+id: sig_qualityofservice
+title: SIG QOS
+---
+
+## Objective
+
+The purpose of this SIG is to allow a RSE handle in an automatic manner storage
+with two different qualitiy of services based on access latency. Initial work in
+this area differentiates between disk and tape within the same RSE. New data files
+are initially pinned on disk either for the lifetime of the rule or up to the 
+maximum set by the storage site. After the pin expires the local storage system is
+free to stage the file to tape for further access. When Rucio needs to access such
+files again, the bring on line command is given to the RSE and the file is 
+transferred from tape to disk if needed. Currently this is being tested with dCache.
+In the future, it can be extended to other storage systems where appropriate.
+At the end of this SIG a report will be written documenting the status of this 
```

Comment:
I wouldn't say status, rather outcome/benefit or something like this.

Reply:
made the change requested